### DISCUSSION TITLE
1.5.4 kade engine master code error help

### DISCUSSION THREAD

**Discussion by glowsoony** [KadeArchive/Kade-Engine#1288] [Q&A]
![image](https://user-images.githubusercontent.com/84847356/125362896-abd05700-e33d-11eb-8873-195effd3c6fc.png)
please need help i bad at some code. the run lime test windows was for something but i didnt mean to put it in the cmd prompt

**Comment by xsolvia** [comment]
just do `lime build windows`

if that doesn't work do `haxelib run lime setup` and then run `lime build windows`

**Reply by glowsoony** [reply]
welp 

![image](https://user-images.githubusercontent.com/84847356/125380660-8d2f8780-e360-11eb-90ce-3df5d8279605.png)
